aspnetcore/web-api/advanced/custom-formatters.md

```diff
@@ -10,9 +10,9 @@ uid: web-api/advanced/custom-formatters
 
 By [Tom Dykstra](https://github.com/tdykstra)
 
-ASP.NET Core MVC supports data exchange in Web APIs using input and output formatters. Input formatters are used by [Model Binding](xref:mvc/models/model-binding); output formatters are used to [format responses](xref:web-api/advanced/formatting).
+ASP.NET Core MVC supports data exchange in Web APIs using input and output formatters. Input formatters are used by [Model Binding](xref:mvc/models/model-binding). Output formatters are used to [format responses](xref:web-api/advanced/formatting).
 
-The framework provides built-in input and output formatters for JSON and XML. It provides a built-in output formatter for plain text, but doesn't currently provide an input formatter for plain text.
+The framework provides built-in input and output formatters for JSON and XML. It provides a built-in output formatter for plain text, but doesn't provide an input formatter for plain text.
```

Comment:
I took this out and put it back a few times. Ryan's comment on the original review used "currently" but I did wonder if that suggested it was going to be added soon.